iac-extensions/kubernetes/src/test/resources/checks/DeprecatedCodeCheck/helm/helm3/templates/_helpers.tpl

```diff
@@ -0,0 +1,11 @@
+{{/*
+Return the appropriate apiVersion for deployment.
+*/}}
+{{- define "deployment.apiVersion" -}}
```

Comment:
The name `deployment.apiVersion` is not the best as `apiVersion` is used in many places and may confuse us. Maybe `deployment.gitVersion` will be better?

Reply:
This function is actually to get the value that will be in the `apiVersion: ...` field of a Deployment. It uses `KubeVersion` to determine a suitable `apiVersion`